aptos-move/framework/aptos-stdlib/sources/string_utils.move

```diff
@@ -1,8 +1,8 @@
 module aptos_std::string_utils {
     use std::string::String;
 
-    const EARGS_MISMATCH: u64 = 1;
-    const EINVALID_FORMAT: u64 = 2;
+    const EARGS_MISMATCH: u64 = 1;  // The number of values in the list does not match the number of "{}" in the format string.
```

Comment:
Please do documentation style (everywhere you add comments).

```
/// This code indicates a problem.
const EPROBLEM_CODE: u64 = 1;
```

In this case, these comments are actually extracted by a tool and put into an error description map which is included in Aptos metadata of a bytecode module. Not sure whether other comment styles would work with this.